### ISSUE TITLE
Gallery: Randomize order: Evoke notification

### ISSUE THREAD

**Issue by hanneslsm** [WordPress/gutenberg#58558]
via https://github.com/WordPress/gutenberg/pull/58407#discussion_r1474384774

## What problem does this address?
The "Random Order" toggle should evoke a notification to give visual feedback to the user.

## What is your proposed solution?
To be consistent with the other notifications, let's add a notification saying
"All gallery images will be displayed in random order" or
"All gallery images will be displayed in random order on each page load"

<img width="382" alt="image" src="https://github.com/WordPress/gutenberg/assets/17861443/6dd94c20-c338-4200-9db1-ca877b9d5bb5">

**Comment by richtabor** [comment]
Perhaps, though I'm not convinced that a snackbar gives more validation than the toggle on/off affordance. In both scenarios, you'll want to see it first hand to believe it — for the first application. After that point you'll trust the action. 

As an aside though, we need to tighten up those snackbars to be clearer.

**Comment by hanneslsm** [comment]
> Perhaps, though I'm not convinced that a snackbar gives more validation than the toggle on/off affordance. In both scenarios, you'll want to see it first hand to believe it — for the first application. After that point you'll trust the action.

Fair point. It would give us the option to inform the user more on what they just have changed though, without writing a long text at the toggle.
